configuration/postgres/postgres.go

```diff
@@ -39,6 +39,7 @@ type ConfigurationStore struct {
 	metadata             metadata
 	client               *pgxpool.Pool
 	logger               logger.Logger
+	configLock           sync.Mutex
 	subscribeStopChanMap sync.Map
```

Comment:
If you use a lock you can now avoid using a `sync.Map` here

Reply:
Removed sync.Map